app/javascript/src/components/Common/Navbar/index.jsx

```diff
@@ -1,14 +1,37 @@
 import React from "react";
 
+import { Home, NeetoInsights, UserCircle } from "@bigbinary/neeto-icons";
+import { Sidebar } from "@bigbinary/neetoui/v2/layouts";
 import { Toastr } from "neetoui";
 import { withRouter } from "react-router-dom";
 
 import authenticationApi from "apis/authentication";
 import { resetAuthTokens } from "apis/axios";
 import { useAuthDispatch } from "contexts/auth";
 
-import AccountDropdown from "./AccountDropdown";
-import NavItem from "./NavItem";
+const navLinks = [
+  {
+    icon: function func() {
+      return <Home size={24} />;
+    },
+    label: "Notes",
+    to: "/notes"
+  },
+  {
+    icon: function func() {
+      return <UserCircle size={24} />;
+    },
+    label: "Contact",
+    to: "/notes"
+  },
+  {
+    icon: function func() {
+      return <NeetoInsights size={24} />;
+    },
+    label: "Settings",
+    to: "/form-elements"
+  }
+];
```

Comment:
This can be moved to a `constants.js` file. While exporting constants from a `constants.js` file, please follow this convention - 
`export const NAV_LINKS`